.github/workflows/_build.yaml

```diff
@@ -39,6 +39,18 @@ jobs:
       name: serverless-operator
       dockerfile: components/operator/Dockerfile
       tags: ${{ needs.compute-tags.outputs.tags }}
+      build-args: |
+        SERVERLESS_CONFIG_VERSION=config/serverless
+
+  build-operator-buildless:
+    needs: compute-tags
+    uses: kyma-project/test-infra/.github/workflows/image-builder.yml@main # Usage: kyma-project/test-infra/.github/workflows/image-builder.yml@main
+    with:
+      name: serverless-operator-buildless
+      dockerfile: components/operator/Dockerfile
+      tags: ${{ needs.compute-tags.outputs.tags }}
+      build-args: |
+        SERVERLESS_CONFIG_VERSION=config/buildless-serverless
```

Comment:
If you'd like to use the `config/buildless-serverless` dir during build you have to add the `!config/buildless-serverless` record to the `components/operator/.dockerignore` file